src/test/unit/parsers/test_utils.py

```diff
@@ -310,3 +310,19 @@ def test_safe_get(d, keys, result_value, default):
 def test_should_scrub_domain(regexes, url, expected):
     Configuration.domains_scrubber = regexes
     assert should_scrub_domain(url) == expected
+
+
+def test_str_to_list():
+    assert str_to_list("a,b,c,d") == ["a", "b", "c", "d"]
+
+
+def test_str_to_list_exception():
+    assert str_to_list([]) is None
```

Comment:
```suggestion
    assert str_to_list("") is None
```
I meant that (because that's how you call it when you define your variables...)